packages/client/src/App.tsx

```diff
@@ -66,11 +66,7 @@ import { UserList } from './views/SysAdmin/Team/user/UserList'
 import { ApplicationConfig } from './views/SysAdmin/Config/Application'
 import { VerifyCorrector } from './views/CorrectionForm/VerifyCorrector'
 import { CorrectionForm, CorrectionReviewForm } from './views/CorrectionForm'
-import {
-  FormConfiguration,
-  FormConfigWizard
-} from './views/SysAdmin/Config/Forms'
-import { FormConfigSettings } from './views/SysAdmin/Config/Forms/FormConfigSettings'
+import { FormConfigHome, FormConfigWizard } from './views/SysAdmin/Config/Forms'
```

Comment:
Relative imports aren't always a bad choice. Rather if we try to use absolute imports everywhere it will make our life more difficult than making it easier